Examples/JavaScript/JavaScript Essentials/Objects and Arrays.ts

```diff
@@ -0,0 +1,96 @@
+//// { order: 1, compiler: { strict: false } }
+
+// JavaScript objects are collections of values wrapped up with named keys.
+
+const userAccount = {
+  name: "Kieron",
+  id: 0
+}
+
+// You can combine these to make larger, more complex data-models.
+
+const pie = {
+  type: "Apple"
+}
+
+const purchaseOrder = {
+  owner: userAccount,
+  item: pie
+}
+
+// If you use your mouse to hover over some of these words (try purchaseOrder above)
+// you can see how TypeScript is interpreting your JavaScript into labeled types.
+
+// Values can be accessed via the ".", so to get a username for a purchase order
+console.log(purchaseOrder.item.type)
+
+// If you hover your mouse over each part of the code between the ()s, you can 
+// see TypeScript offering more information about each part. Try re-writing this below:
+
+// Copy this in the next line, character by character: purchaseOrder.item.type
+
+
+
+// TypeScript provides feedback to the Playground about what JavaScript objects 
```

Comment:
I would put this, or part of it, before the "Copy this in the next line" to make it clear to expect completions. (The transition from quick info in the previous paragraph wasn't obvious to me.)